src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -73,14 +83,15 @@ class ReflectionExtractor implements PropertyListExtractorInterface, PropertyTyp
      * @param string[]|null $accessorPrefixes
      * @param string[]|null $arrayMutatorPrefixes
      */
-    public function __construct(array $mutatorPrefixes = null, array $accessorPrefixes = null, array $arrayMutatorPrefixes = null, bool $enableConstructorExtraction = true, int $accessFlags = self::ALLOW_PUBLIC, InflectorInterface $inflector = null)
+    public function __construct(array $mutatorPrefixes = null, array $accessorPrefixes = null, array $arrayMutatorPrefixes = null, bool $enableConstructorExtraction = true, int $accessFlags = self::ALLOW_PUBLIC, InflectorInterface $inflector = null, int $magicMethodsFlags = self::ALLOW_MAGIC_GET | self::ALLOW_MAGIC_SET)
```

Comment:
The `magicMethodsFlags` defaults are configured here for BC with 5.1, but before 5.1, magic get and set weren't supported, right?
So should it have been considered a BC break to enable them by default on 5.1?

Reply:
@ogizanagi Yes, this looks like a BC break to me. I think we need to set the default value to `ReflectionExtractor::DISALLOW_MAGIC_METHODS`. We can consider deprecating this default value, but I am not convinced that's really necessary.